pokerbot.js

```diff
@@ -144,6 +167,30 @@ pokerbot.root = function (req, res, next) {
     console.log('Option start : end')
     return res.status(200).json(response)
   }
+
+  // Get the status of game
+  if (option === 'status' && (pokerbot.pokerDataModel.hasOwnProperty(jiraId))) {
```

Comment:
This function is *huge* and should be refactored. A function should do *one thing*. This function does at least six things. Each of the sections like this should be factored out into another function and this main function should call them. Another rule of thumb is that any one function should be at most 50 lines. Please clean this function up.